app/build.gradle

```diff
@@ -65,14 +65,10 @@ dependencies {
     Once build and published, we can test the uploaded library by referring to the repository instead
      */
 
-//    def emdk_version = "1.0.0"
+//    def emdk_version = "1.0.1"
 //    implementation "com.github.gls-denmark:kotlin-emdk:$emdk_version"
```

Comment:
If you have this in the documentation, I think you can delete it from here, so you do not have to update it each time.

Reply:
I have linked to jitpack so users can see what the latest version is and use that instead of us having to bump them each time